services/app-api/src/handlers/apollo_gql_express.ts

```diff
@@ -0,0 +1,144 @@
+app.use(cors({ origin: true, credentials: true }))
+app.use(express.json({ limit: '10mb' }))
+
+// --- Apollo Server setup ---
+async function buildApolloServer() {
+    // These env vars and config mirror your current handler
+    const stageName = process.env.stage || 'local'
+    const dbURL = process.env.DATABASE_URL || ''
+    const secretsManagerSecret = process.env.SECRETS_MANAGER_SECRET || ''
+    const applicationEndpoint = process.env.APPLICATION_ENDPOINT || ''
+    const emailerMode = process.env.EMAILER_MODE || 'LOCAL'
+    const parameterStoreMode = process.env.PARAMETER_STORE_MODE || 'LOCAL'
+    const ldSDKKey = process.env.LD_SDK_KEY || ''
+    if (!process.env.JWT_SECRET) {
+        throw new Error('JWT_SECRET environment variable is required but not set.')
+    }
+    const jwtSecret = process.env.JWT_SECRET
+
+    // Postgres
+    const pgResult = await configurePostgres(dbURL, secretsManagerSecret)
+    if (pgResult instanceof Error) throw pgResult
+    const store = NewPostgresStore(pgResult)
+
+    // LaunchDarkly
+    let launchDarkly
+    try {
+        const ldClient = ld.init(ldSDKKey, {})
+        await ldClient.waitForInitialization({ timeout: 10 })
+        launchDarkly = ldService(ldClient)
+    } catch {
+        launchDarkly = offlineLDService()
+    }
+
+    // Apollo plugins
+    let plugins = []
+    let introspectionAllowed = false
+    if (stageName === 'prod' || stageName === 'val') {
+        plugins = [ApolloServerPluginLandingPageDisabled()]
+    } else {
+        plugins = [ApolloServerPluginLandingPageLocalDefault({ embed: true })]
+        introspectionAllowed = true
+    }
+
+    // JWT
+    const jwtLib = newJWTLib({
+        issuer: `mcreview-${stageName}`,
+        signingKey: Buffer.from(jwtSecret, 'hex'),
+        expirationDurationS: 90 * 24 * 60 * 60,
+    })
+
+    // Email parameter store
+    const emailParameterStore =
+        parameterStoreMode === 'LOCAL'
+            ? newLocalEmailParameterStore()
+            : newAWSEmailParameterStore()
+    const emailer = await configureEmailer({
+        emailParameterStore,
+        store,
+        ldService: launchDarkly,
+        stageName,
+        emailerMode,
+        applicationEndpoint,
+    })
+    if (emailer instanceof Error) throw emailer
+
+    // S3ClientT is required by configureResolvers, but not needed for this prep file.
+    // Provide a dummy object to satisfy the type.
+    const dummyS3Client = {} as S3ClientT
+
+    // Resolvers
+    const resolvers = configureResolvers(
+        store,
+        emailer,
+        launchDarkly,
+        jwtLib,
+        dummyS3Client, // Satisfy type requirement
+        applicationEndpoint
+    )
+
+    // Context
+    // const userFetcher =
+    //   authMode === 'LOCAL' ? userFromLocalAuthProvider : userFromCognitoAuthProvider;
+    const contextForRequest = () => ({ user: { id: 'placeholder' } }) // Replace with your real context logic if needed
```

Comment:
Do we have a ticket in this epic to bring over user auth, or do we want to finish that here?

Reply:
I'll add a ticket for that